### PR TITLE
Firebase config automation

### PR DESCRIPTION
**Pull request by mjovkovic**

### CODE REVIEW

File: lib/server/server.js [hunk 1]
```diff
@@ -298,9 +318,99 @@ function convertApnToFcmToken(appName, tokenObject) {
...
+      return;
+    }
+
+    listApps(projectManagement, platform).then(apps => {
+
+      if (!apps || apps.length === 0) {
+        createApp(projectManagement, vet, platform)
+          .then(app => {
+            if (!app) {
+              reject(new Error(`Error creating android app for ${vetId}`));
```
Comment (mslobodan) [inline lib/server/server.js]: @mjovkovic Is `return` missing here?
```suggestion
              reject(new Error(`Error creating ${platform} app for ${vetId}`));
              return;
```

File: lib/server/server.js [hunk 2]
```diff
@@ -298,9 +318,99 @@ function convertApnToFcmToken(appName, tokenObject) {
...
+            }
+
+            getConfigForApp(app, resolve, reject);
```
Comment (mslobodan) [inline lib/server/server.js]: @mjovkovic `getConfigForApp` will resolve or reject promise from `configureApp` which is weird. I think we should remove `getConfigForApp` function, and use its logic directly in `configureApp` function.

File: lib/server/server.js [hunk 3]
```diff
@@ -298,9 +318,99 @@ function convertApnToFcmToken(appName, tokenObject) {
...
+          })
+          .catch(error => {
+            reject(error);
+          });
+      } else if (apps.length === 1) {
+        getConfigForApp(apps[0], resolve, reject);
+      } else {
+        console.log('PUSH: Multiple android apps found', apps);
```
Comment (mslobodan) [inline lib/server/server.js]: @mjovkovic 
```suggestion
        console.log('PUSH: Multiple ${platform} apps found', apps);
```

**Review by mslobodan — CHANGES_REQUESTED** [review]

File: lib/server/server.js (hunk 1 above)
Comment (mjovkovic) [reply lib/server/server.js]: @mslobodan nice catch, there should be `return` after reject.

File: lib/server/server.js (hunk 2 above)
Comment (mjovkovic) [reply lib/server/server.js]: @mslobodan I agree

**Review by mslobodan — APPROVED** [review]

**Pull request merged**